extensions/amp-video/0.1/amp-video.js

```diff
@@ -199,6 +200,9 @@ class AmpVideo extends AMP.BaseElement {
     this.configure_();
 
     this.video_ = element.ownerDocument.createElement('video');
+    if (this.element.hasAttribute('multi-bitrate')) {
```

Comment:
Can we detect this with a querySelector instead? I'm afraid of publisher misconfiguration, providing multiple bitrates but never realizing that they're not used.

Reply:
My idea was to enforce this in the validator. Also, this attribute makes this invalid and hence safe to land for now.

Reply:
Actually, I see your point. I think this can land for now. We'd otherwise need a check that not just one child but all children have a bitrate supplied.

Reply:
We need to handle sources without a bitrate no matter what. I don't think you can write a validation rule that requires an attribute based on its descendants?
I still think we'd reduce risks of misconfiguration with a simple `source[data-bitrate]` querySelector.

Reply:
Done

Reply:
One thing to consider. Right now this code assumes that unknown bitrate is 0. One could also make the case for Infinity. Ideally, of course, all sources have a data-bitrate attribute

Reply:
Actually, I changed my mind and made it infinity. I think that if people forget, then that source if most likely the least compressed version.